kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/servlets/KmeliaRequestRouter.java

```diff
@@ -375,8 +376,13 @@ public String getDestination(String function, KmeliaSessionController kmelia,
       } else if ("ToPublicationHeader".equals(function)) {
         String action = (String) request.getAttribute("Action");
         if ("UpdateView".equals(action)) {
-          request.setAttribute("AttachmentsEnabled", false);
           request.setAttribute("TaxonomyOK", kmelia.isPublicationTaxonomyOK());
+          List<ClassifyPosition> positions = kmelia.getSessionPubliOrClone().getPDCPositions();
+          boolean isPublicationClassifiedOnPDC = true;
+          if (positions.isEmpty())
+            isPublicationClassifiedOnPDC = false;
```

Comment:
A Java rule: the body of the if and else-statements has to be put in brackets, even there is a single statement of code

Reply:
Done.